eZ/Publish/Core/FieldType/Image/ImageStorage/Gateway/DoctrineStorage.php

```diff
@@ -313,4 +345,80 @@ public function extractFilesFromXml($xml)
 
         return null;
     }
+
+    public function getAllImagesData(): array
+    {
+        $selectQuery = $this->connection->createQueryBuilder();
+        $selectQuery
+            ->select(
+                $this->connection->quoteIdentifier('contentobject_attribute_id'),
+                $this->connection->quoteIdentifier('filepath')
+            )->from($this->connection->quoteIdentifier(self::IMAGE_FILE_TABLE))
```

Comment:
`from` should use aliases
```suggestion
            ->select(
                'img.contentobject_attribute_id',
                'img.filepath'
            )->from(self::IMAGE_FILE_TABLE, 'img')

```